### PR TITLE
Update PDR spec tests to use unique granule IDs

### PR DESCRIPTION
**Pull request by Jkovarik**
Updates integration tests to use unique granuleIds for all PDR tests

### CODE REVIEW

File: example/spec/helpers/testUtils.js [hunk 1]
```diff
@@ -50,26 +50,46 @@ function templateFile({ inputTemplateFilename, config }) {
...
- * @param {string} prefix - S3 folder prefix
- * @param {Array<StringReplacement>} [replacements] - array of replacements in file content e.g. [{old: 'test', new: 'newTest' }]
+ * @param {Object} params                         - parameters
+ * @param {string} params.file                    - filename of data to upload
+ * @param {string} params.bucket                  - bucket to upload to
+ * @param {string} params.prefix                  - S3 folder prefix
+ * @param {string} params.targetReplacementRegex  - regexp to allow file copy target to target
+ *                                                  a different s3 key from the original file
+ *                                                  if specified will replace matches with
+ *                                                  targetReplacementString
```
Comment (botanical) [inline example/spec/helpers/testUtils.js:60]: Hm this description is a bit confusing to me 🤔  Did you mean to put something like:
```
regexp to allow file copy target to target
a different s3 key from the original file
if specified 
```
?

Comment (Jkovarik) [reply example/spec/helpers/testUtils.js]: Yeah, thanks, that *was* a bit mangled.

Comment (Jkovarik) [reply example/spec/helpers/testUtils.js]: e4bc5ad

File: example/spec/parallel/ingest/ingestFromPdrSpec.js [hunk 2]
```diff
@@ -105,26 +110,29 @@ describe('Ingesting from PDR', () => {
...
-          config.bucket,
-          unmodifiedS3Data,
-          testDataFolder
-        ),
-        addCollections(config.stackName, config.bucket, collectionsDir, testSuffix, testId),
-        addProviders(config.stackName, config.bucket, providersDir, config.bucket, testSuffix),
-      ]);
-
-      addedCollection = populatePromises[2][0];
+      [addedCollection] = await Promise.all(
```
Comment (Jkovarik) [inline example/spec/parallel/ingest/ingestFromPdrSpec.js:113]: This won't work!    Need to fix/bad refactor

Comment (Jkovarik) [reply example/spec/parallel/ingest/ingestFromPdrSpec.js:113]: @botanical  apologies, missed this on the last cleanup commit 😓  77744938f

File: example/spec/parallel/ingest/ingestFromPdrSpec.js [hunk 3]
```diff
@@ -14,13 +14,16 @@
  * pdr status check
  * This will kick off the ingest workflow
  *
- * Ingest worklow:
+ * Ingest workflow:
```
Comment (botanical) [inline example/spec/parallel/ingest/ingestFromPdrSpec.js:17]: ⭐

**Review by botanical — APPROVED** [review]
Thanks for addressing my feedback! Pulled this down and tested it on my stack and looks good to me.

**Pull request merged**